<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>166</width>
      <height>25</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QFrame" name="game_frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_rooms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_field">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="red">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
color: red;</string>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orange">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange;
color: orange;</string>
    </property>
    <property name="text">
     <string>orange</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellow">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow;
color: yellow;</string>
    </property>
    <property name="text">
     <string>yellow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="green">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
color: green;</string>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: cyan;
color: cyan;</string>
    </property>
    <property name="text">
     <string>cyan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dark_blue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;
color: blue;</string>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purple">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: purple;
color: purple;</string>
    </property>
    <property name="text">
     <string>purple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="white">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: white;</string>
    </property>
    <property name="text">
     <string>white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="black">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color: black;</string>
    </property>
    <property name="text">
     <string>black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pic">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
